### ISSUE TITLE
[Bug] _reactNativeMmkvStorage.create is not a function

### ISSUE THREAD

**Issue by MasterMedo** [ammarahm-ed/react-native-mmkv-storage#116]
**Describe the bug**
I added mmkv-storage to a blank project and followed the instructions given [here](https://rnmmkv.vercel.app/#/usemmkvstorage). When I run the app with expo it says `create` is not a function (I do have mmkv-storage version 0.5.9). Adding the hook manually like in previous versions works as expected.

**To Reproduce**
1. `expo init` -> "blank TypeScript react native app"
2. `npm install react-native-mmkv-storage`
3. check mmkv version in package json, should be '0.5.9' or higher
4. Add this to `App.tsx`:
```jsx
import MMKVStorage, {create} from "react-native-mmkv-storage";
const MMKV = new MMKVStorage.Loader().initialize();

export const useStorage = create(MMKV);
```
5. `expo start`
6. See error
```
TypeError: (0, _reactNativeMmkvStorage.create) is not a function. (In '(0, _reactNativeMmkvStorage.create)(MMKV)', '(0, _reactNativeMmkvStorage.create)' is undefined)
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:104:6 in reportException
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:171:19 in handleException
at node_modules/react-native/Libraries/Core/setUpErrorHandling.js:24:6 in handleError
at node_modules/expo-error-recovery/build/ErrorRecovery.fx.js:12:21 in ErrorUtils.setGlobalHandler$argument_0
at [native code]:null in flushedQueue
at [native code]:null in invokeCallbackAndReturnFlushedQueue
```

**Expected behavior**
No error happens.

**Screenshots**
![image](https://user-images.githubusercontent.com/16375100/120917274-a3399080-c6ae-11eb-8071-79d02329ac74.png)

**Platform Information:**
 - OS: iOS
 - React Native version: `https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz`
 - mmkv storage version: `^0.5.9`

**Comment by ammarahm-ed** [comment]
The issue is fixed on master. A new version will come soon.

**Comment by ammarahm-ed** [comment]
Fixed in v0.6.0
